subprojects/model-core/src/main/java/org/gradle/internal/reflect/annotations/impl/DefaultTypeAnnotationMetadataStore.java

```diff
@@ -127,6 +130,7 @@ public DefaultTypeAnnotationMetadataStore(
         this.globallyIgnoredMethods = allMethodsOf(ignoreMethodsFromTypes);
         this.mutableNonFinalClasses = ImmutableSet.copyOf(mutableNonFinalClasses);
         this.ignoredMethodAnnotations = ImmutableSet.copyOf(ignoredMethodAnnotations);
+        this.propertyTypeAnnotations = ImmutableSet.copyOf(propertyTypeAnnotations);
```

Comment:
@bamboo  I am reworking this file, as this breaks a few tests. Please ignore it for now.

Reply:
Good to go now